sdk/src/main/scala/com/horizen/SidechainMemoryPool.scala

```diff
@@ -6,17 +6,23 @@ import com.horizen.node.NodeMemoryPool
 import com.horizen.transaction.BoxTransaction
 import com.horizen.utils.FeeRate
 import scorex.util.ModifierId
+import scorex.util.{ModifierId, ScorexLogging}
 import scorex.core.transaction.MempoolReader
 
 import scala.collection.concurrent.TrieMap
 import scala.util.{Failure, Success, Try}
 import scala.collection.JavaConverters._
 
-class SidechainMemoryPool(unconfirmed: TrieMap[String, SidechainMemoryPoolEntry])
+class SidechainMemoryPool(unconfirmed: TrieMap[String, SidechainMemoryPoolEntry], mempoolSettings: MempoolSettings)
   extends scorex.core.transaction.MemoryPool[SidechainTypes#SCBT, SidechainMemoryPool]
   with SidechainTypes
   with NodeMemoryPool
+  with ScorexLogging
 {
+  var maxPoolSizeBytes : Long =  mempoolSettings.maxSize * 1024 * 1024
+  var usedPoolSizeBytes : Long = 0
```

Comment:
why `0`?
I guess we may take in consideration the `unconfirmed` (can be not empty)